### PR TITLE
Add licence agreement in Greek in README and create TRANSLATORS file

### PR DESCRIPTION
**Pull request by lysnikolaou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
 # python-docs-gr
 Greek Translation of the Python Documentation
+
```
Comment (thepetk) [inline README.md:3]: I like that we have the agreement in greek. Do you think we should also add the original English text?

cc @lysnikolaou

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+### Σημείωμα για την άδεια χρήσης της μετάφρασης
```
Comment (thepetk) [inline README.md:4]:
```suggestion
### Συμφωνητικό Συνεισφοράς στην Μετάφραση των Εγγράφων

ΣΗΜΕΙΩΣΗ ΓΙΑ ΤΗΝ ΑΔΕΙΑ ΧΡΗΣΗΣ ΤΩΝ ΜΕΤΑΦΡΑΣΕΩΝ
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+
+Ο οδηγός χρήσης της Python συντηρείται από ένα παγκόσμιο δίκτυο
+εθελοντών. 
+
+Αυτό το έργο πρότζεκτ μια συνεργασία ανάμεσα σε εθελοντές μεταφραστές και
```
Comment (thepetk) [inline README.md:9]:
```suggestion
Αυτό το έργο πρότζεκτ αποτελεί μια συνεργασία ανάμεσα σε εθελοντές μεταφραστές και
```

File: TRANSLATORS [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Fanis Petkos
```
Comment (thepetk) [inline TRANSLATORS:1]:
```suggestion
Theofanis Petkos
```
Let's put the official name here :D

File: README.md (hunk 1 above)
Comment (thepetk) [reply README.md:3]:
```suggestion

### Documentation Contribution Agreement

NOTE REGARDING THE LICENSE FOR TRANSLATIONS: Python's documentation is maintained using a global network of volunteers. By posting this project on Transifex, GitHub, and other public places, and inviting you to participate, we are proposing an agreement that you will provide your improvements to Python's documentation or the translation of Python's documentation for the PSF's use under the CC0 license (available at `https://creativecommons.org/publicdomain/zero/1.0/legalcode`_). In return, you may publicly claim credit for the portion of the translation you contributed and if your translation is accepted by the PSF, you may (but are not required to) submit a patch including an appropriate annotation in the Misc/ACKS or TRANSLATORS file. Although nothing in this Documentation Contribution Agreement obligates the PSF to incorporate your textual contribution, your participation in the Python community is welcomed and appreciated.

You signify acceptance of this agreement by submitting your work to the PSF for inclusion in the documentation.

```

**Review by thepetk — CHANGES_REQUESTED** [review]
Left some suggestions. One minor comment is if we could make each paragraph one line? WDYT?

File: README.md (hunk 1 above)
Comment (lysnikolaou) [reply README.md:3]: Hmm, not sure. Most of the translations I checked only include it in their respective language. Maybe just add a link to [the original text](https://peps.python.org/pep-0545/#setup-the-documentation-contribution-agreement) below the Greek version?

**Comment by lysnikolaou** [comment]
> One minor comment is if we could make each paragraph one line?

Whatever you prefer. I'm not really a fan of one paragraph per line, but I don't have a strong opinion.

File: README.md (hunk 1 above)
Comment (thepetk) [reply README.md:3]: Yes I think that works too!

**Comment by thepetk** [comment]
> > One minor comment is if we could make each paragraph one line?
> 
> Whatever you prefer. I'm not really a fan of one paragraph per line, but I don't have a strong opinion either way.
> 

Lets keep it as is because I dont have a strong opinion either!

**Pull request merged**